server/src/helpers/translation.js

```diff
@@ -0,0 +1,65 @@
+const key = process.env.CLOUD_TRANSLATION_API_KEY || ''
+
+const detectLanguage = async text => {
+  try {
+    const response = await fetch(
+      `https://translation.googleapis.com/language/translate/v2/detect?key=${key}`,
+      {
+        method: 'POST',
+        body: JSON.stringify({
+          q: text
+        })
+      }
+    )
+    const res = await response.json()
+    const data = await res.data
+    return data.detections[0][0].language
```

Comment:
`res.data` ne peut pas être une promesse donc pas besoin de await ici
```suggestion
    return res.data.detections[0][0].language
```

A faire dans `getTranslatedText` aussi.